css/style.css

```diff
@@ -1,3 +1,183 @@
-h1 {
-  color: salmon;
+body{
+  font-family: "LL brown", Arial, sans-serif ;
+  background-color: rgb(237,239,236);
+  color: #55595D;
+  font-weight: normal;
+
 }
+/*main file*/
+main{
+  margin: 0 auto;
+  width: 1200px;
+  text-align: center;
+  letter-spacing: 2px;
+}
+h2{
+  font-size: 32px;
+  font-weight: lighter;
+}
+h3{
+  font-size: 18px;
+  padding-bottom: 10px;
+  font-weight: normal;
+}
+
+/* Logo*/
+.logo{
+  margin: 0;
+  padding: 0;
+}
+.logo img{
+  width: 80px;
+  height: 80px;
+}
+
+/*navigation*/
+nav{
+  display: flex;
+  flex-flow: row;
+  justify-content: space-between;
+}
+.nav_list ul{
+  display: block;
+  margin: 0;
+  padding:0;
+  list-style-type: none;
+}
+
+.nav_list li{
+  text-decoration: none;
+  text-align: center;
+  float: left;
+  margin-top: 20px;
+  padding-right: 20px;
+
+}
+
+.nav_list a{
+  display: inline-block;
+  text-decoration: none;
+  color:#000;
+  font-size: 20px;
+  padding-right: 20px;
+}
+
+/*extra link-Guidebooks and weekend*/
+
+div.Guidebooks>div{
```

Comment:
Yes, thanks. I usually focus first on the functionality and forget a lot of formatting and common practice. I guess I have to do it parallel.